index.html

```diff
@@ -1,1632 +0,0 @@
-<!doctype html><html lang="en">
```

Comment:
You can probably add a `.gitignore` to avoid adding the html output to the repo.